.gitignore

```diff
@@ -3,3 +3,7 @@
 *.tar.gz
 /tuareg.*/
 /opam/url
+commit.txt
+RCS/
+fehler.txt
```

Comment:
Am 21.06.2015 um 23:24 schrieb Christophe Troestler:
>
> In .gitignore 
> <https://github.com/ocaml/tuareg/pull/54#discussion_r32897800>:
>
> > @@ -3,3 +3,7 @@
> >  *.tar.gz
> >  /tuareg.*/
> >  /opam/url
> > +commit.txt
> > +RCS/
> > +fehler.txt
>
> What are these 3 files good for?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/ocaml/tuareg/pull/54/files#r32897800>.
>

Please edit these minor changes as you need it.